components/prism-autohotkey.js

```diff
@@ -1,16 +1,20 @@
 // NOTES - follows first-first highlight method, block is locked after highlight, different from SyntaxHl
 Prism.languages.autohotkey= {
-	'comment': {
-		pattern: /(^[^";\n]*("[^"\n]*?"[^"\n]*?)*)(?:;.*$|^\s*\/\*[\s\S]*\n\*\/)/m,
-		lookbehind: true
-	},
+	'comment': [
+		{
+			pattern: /(^|\s);.*/m,
```

Comment:
You don't need the `m` flag.
`\s` will also match `\n` and `\r`. Do keep `^` though.